src/services/api/categoriesService.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+const baseUrl = process.env.REACT_APP_CATEGORIES_BASE_URL;
```

Comment:
we have to ensure that in varcel we have this variable, please reach out to Mateusz

Reply:
I don't know yet if it's possible to run react app & json-server on vercel